server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -269,6 +269,7 @@
                 /api/feeds/**=ROLE_SUPERVISOR
                 /api/support=ROLE_SUPERVISOR
                 /api/pipelines.xml=ROLE_USER
+                /api/server_version=ROLE_USER
```

Comment:
You probably mean to say `IS_AUTHENTICATED_ANONYMOUSLY` so that anonymous users can call this too?